CHANGES

```diff
@@ -158,7 +154,16 @@ Notations
   Note that this might cause incompatibilities if you have, e.g., list_scope
   and vector_scope both open with vector_scope on top, and expect `++` to
   refer to `app`.
-  Solution: wrap `_ ++ _` in `(_ ++ _)%list` (or whichever scope you want). 
+  Solution: wrap `_ ++ _` in `(_ ++ _)%list` (or whichever scope you want).
+
+Changes from 8.8.1 to 8.8.2
+===========================
+
+Tools
+
+- The targets `print-pretty-timed`, `print-pretty-timed-diff`, and
```

Comment:
Note that in the version I backported I reworded to "The coq-makefile targets" to be clearer without context.

Reply:
Thanks, updated